eval/adjust.rkt

```diff
@@ -64,302 +64,50 @@
       (or (not useful?)
           (and (<= prec-new prec-old)
                (andmap (lambda (x) (or (< x varc) (vector-ref vrepeats (- x varc)))) (cdr instr)))))
-    (set! any-false? (or any-false? (not repeat)))
     (vector-set! vrepeats n repeat))
 
   ; Step 4. Copying new precisions into vprecs
-  (vector-copy! vprecs 0 vprecs-new)
-
-  ; Step 5. If precisions have not changed but the point didn't converge. A problem exists - add slack to every op
-  (unless any-false?
-    (set-rival-machine-bumps! machine (add1 bumps))
-    (define slack (get-slack))
-    (for ([prec (in-vector vprecs)] [n (in-range (vector-length vprecs))])
-      (define prec* (min (*rival-max-precision*) (+ prec slack)))
-      (when (equal? prec* (*rival-max-precision*))
-        (*sampling-iteration* (*rival-max-iterations*)))
-      (vector-set! vprecs n prec*))
-    (vector-fill! vrepeats #f)))
```

Comment:
We never use this step. Probably can be removed

Reply:
I believe that it can be the case, another thing I am thinking off is overflows/underflows, where tuning does not help anymore (ex: `(log (exp x))`). Such cases are supposed to be rare since for the most of the overflows/underflows we can actually prove that the precision is not reachable. However, here, given that `x` is exact we can not prove that. 
I can give nightly a run with `bumps` turned on again and we will check whether it affects too much